christianR15c/chat-room-back-end
# room & user crud operations

# ft crud operation of room & user
## What is about this PR
- admin can be able to create rooms
- user can be able to sign up

## Why 
- they must be rooms & users so that in the future admin will be able to assign users to certain rooms

## How to test
- clone this branch if you haven't done it yet
- checkout to this branch by typing this command `git checkout ft-crud-room-user`
- install all dependencies by typing this command `npm install` or `npm i`
- open your code editor terminal and run `npm run dev` then you should be able to see that the server is running 
- open postman to test endpoints
- to create room `api/v1/room/`, to update `api/v1/room/{roomId}`, to get single room `api/v1/room/{roomId}`, to delete use `api/v1/room/{roomId}
- - to create user `api/v1/user/`, to update `api/v1/user/{userId}`, to get single room `api/v1/user/{userId}`, to delete use `api/v1/user/{userId}
